Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -989,7 +989,7 @@ theorem fundamentalSequenceProp_inr (o f) :
   Iff.rfl
 
 theorem fundamentalSequence_has_prop (o) : FundamentalSequenceProp o (fundamentalSequence o) := by
-  induction' o with a m b iha ihb; · exact rfl
+  induction o with | zero => exact rfl | oadd a m b iha ihb
```

Comment:
I don't like that the cases are all on one line, which hides the case that we are really interested in.
```suggestion
  induction o with
  | zero => exact rfl
  | oadd a m b iha ihb
```

Reply:
Fixed.